### DISCUSSION TITLE
About localization

### DISCUSSION THREAD

**Discussion by hyi1233** [mriscoc/Ender3V2S1#103] [Q&A] [answered]
I see the system menu, there is a localization option, whether to support localization translation, how to do!

**Comment by mriscoc** [comment, marked as answer]
https://github.com/mriscoc/Ender3V2S1/blob/5562e7535c84155f3ec359b1b9f776ef1a0499ff/Marlin/Configuration.h#L2097

**Reply by hyi1233** [reply]
Why did the compilation fail? I'm a green hand. I can't really C++
> Executing task: c:\Users\cq520wl\Desktop\Ender3V2S1\platformio.exe run <
